README.md

```diff
@@ -150,18 +189,29 @@ tasks.test {
 ```groovy
 tasks.test {
     kover {
-        disabled = false // true to disable instrumentation tests of this task, Kover reports will not depend on the results of their execution 
-        binaryReportFile.set(file("$buildDir/custom/result.bin")) // set file name of binary report
-        includes = ['com.example.*'] // see "Instrumentation inclusion rules" below
-        excludes = ['com.example.subpackage.*'] // see "Instrumentation exclusion rules" below
+        // set to true to disable instrumentation of this task, 
+        // Kover reports will not depend on the results of its execution   
+        disabled = false
+      
+        // set file name of binary report
+        binaryReportFile.set(file("$buildDir/custom/result.bin"))
+      
+        // for details, see "Instrumentation inclusion rules" below
+        includes = ['com.example.*']
+      
+        // for details, see "Instrumentation exclusion rules" below
+        excludes = ['com.example.subpackage.*']
```

Comment:
Here and in similar places it's just being nice to the reader so that they don't have to side scroll. 

I didn't touch any of the big blocks with settings as it's taking quite a bit of time, but I would recommend to do the same thing there. Just open the main configuration section and try to read some longer descriptions in the beginning - you'll need to scroll right, but the scroll bar is hidden below and so you need to scroll down first, then scroll right, then scroll up again and only then you can read the description.